src/Trigger/RabbitMQTriggerAttribute.cs

```diff
@@ -9,24 +9,39 @@ namespace Microsoft.Azure.WebJobs
     [Binding]
     public sealed class RabbitMQTriggerAttribute : Attribute
     {
-        public RabbitMQTriggerAttribute(string hostName, string queueName)
+        public RabbitMQTriggerAttribute(string connectionString, string queueName)
+        {
+            this.ConnectionStringSetting = connectionString;
+            this.QueueName = queueName;
+        }
+
+        public RabbitMQTriggerAttribute(string queueName)
         {
-            this.HostName = hostName;
             this.QueueName = queueName;
-            this.BatchNumber = 1;
         }
 
-        public RabbitMQTriggerAttribute(string hostName, string queueName, ushort batchNumber)
+        public RabbitMQTriggerAttribute(string hostName, string userName, string password, int port, string queueName)
         {
             this.HostName = hostName;
+            this.UserName = userName;
+            this.Password = password;
+            this.Port = port;
             this.QueueName = queueName;
-            this.BatchNumber = batchNumber;
+            this.BatchNumber = 1;

```

Comment:
Can BatchNumber be set by users?

Reply:
I used to have an overloaded constructor that takes in batch number but removed it since we don't know how we want to handle batching yet. I can remove anything related to batching from this PR if that makes more sense, since right now we default to one message per batch no matter what